pytest/lib/branches.py

```diff
@@ -38,7 +42,42 @@ def latest_rc_branch():
     if not releases:
         return None
 
-    return semver.VersionInfo.parse(releases[0].title).finalize_version()
+    return str(semver.VersionInfo.parse(releases[0].title).finalize_version())
```

Comment:
@chefsale @bowenwang1996 is it intended the use of `finalize_version` here? Notice it trims any prerelease information from the tag. As I understand `latest_rc_branch` should return branch used for current `testnet`.

Reply:
It is intentional as the branches dont have prerelease parts.